docs/source/modules/event.rst

```diff
@@ -0,0 +1,99 @@
+.. _event-module:
+
+============
+Event Module
+============
+
+.. contents::
+    :depth: 3
+
+Description
+===========
+
+The Event module allows to create listener and assign action to event. The module is using :code:`MotechEvent` to provide
+event with specific data that can be sent by any MOTECH module (using :code:`EventRelay`) when an event is fired.
+The module is using ActiveMQ to support the event execution. It is possible to register your own callback
+function using :code:`EventCallbackService`.
+
+MotechEvent
+===========
+
+The Event module use :code:`MotechEvent` to store information about fired event. Instance of this class can be sent by
+any module to a specific handler to fire an event. Every module can also be a subscriber of an event and receiving events to fire.
```

Comment:
to handle, I guess? If we receive an event, it means that someone else has fired it and we are only receiving it to handle